src/platform/user/profile/vet360/containers/AddressValidationView.jsx

```diff
@@ -31,7 +31,21 @@ import { getValidationMessageKey } from '../../utilities';
 import { ADDRESS_VALIDATION_MESSAGES } from '../../constants/addressValidationMessages';
 
 class AddressValidationView extends React.Component {
+  componentDidUpdate(prevProps) {
```

Comment:
The duplication in life cycle methods between this component and `VAPEditView` make these superb candidates for hooks! 😄  Would be a fun future refactor.

Reply:
Oh I'd love to finally write our own custom hook that wraps up this kind of logic. I haven't done that sort of thing yet.